token/sdk/vault/db/vault.go

```diff
@@ -74,7 +69,7 @@ func (v *VaultProvider) Vault(network string, channel string, namespace string)
 	if err != nil {
 		return nil, errors.WithMessagef(err, "failed to get token db")
 	}
-	tokenStore, err := tokens.NewTokenStore(notifier, tokenDB, tmsID)
+	tokens, err := tokens.Get(v.sp, tmsID)
```

Comment:
I think tokenStore helps more to give context. and `tokens.GetStore` or something like this.

Reply:
this is the token service, though. Shall I call GetService?

Reply:
yes that would make more sense (and be more consistent with other similar cases we have in the code)

Reply:
👍